src/software/thunderscope/field/world_layer.py

```diff
@@ -15,6 +15,7 @@
 from software.thunderscope.thread_safe_buffer import ThreadSafeBuffer
 
 MAX_ALLOWED_KICK_SPEED_M_PER_S = 6.5
+NUM_ROBOTS_ON_FIELD_PER_TEAM = 6
```

Comment:
why do we need this constant?

Reply:
good question, I don't even remember adding it in - removed now